apps/modernization-api/src/test/java/gov/cdc/nbs/patient/profile/CountryCodeResolverTest.java

```diff
@@ -0,0 +1,20 @@
+package gov.cdc.nbs.patient.profile;
+
+import gov.cdc.nbs.entity.odse.PostalLocator;
+import gov.cdc.nbs.entity.srte.CountryCode;
+import org.junit.jupiter.api.Test;
+import java.util.Optional;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+class CountryCodeResolverTest {
+    @Test
+    void should_resolve_state_code_for_postal_locator() {
```

Comment:
This test and `StateCodeResolverTest` are failing with null pointers.  The repositories aren't being set.  We might be able to test these with Cucumber or can pass in mocked repositories.